WordPress/Classes/Utility/ReachabilityUtils.m

```diff
@@ -5,59 +5,6 @@
 @import WordPressUI;
 
 
-@interface ReachabilityAlert : NSObject
```

Comment:
Since the "alerts" initiated from `ReachabilityUtils` were removed, this is no longer necessary.